packages/generator-wcfactory/generators/element/templates/package.json

```diff
@@ -41,6 +43,11 @@
     <%- libraryDependencies %>
   },
   "devDependencies": {
+    "@open-wc/testing": "2.5.32",
+    "@web/test-runner": "0.12.19",
+    "@web/test-runner-commands": "0.4.2",
+    "@web/test-runner-playwright": "0.8.4",
+    "@web/test-runner-puppeteer": "0.9.3",
```

Comment:
So what we want is `packages/generator-wcfactory/generators/start/templates/templates/libraries` and the included package.json files in THIS location (4 of them by default I believe) to be where this gets modified.